app-crypt/efitools/files/efitools-1.8.1-openssl-1.1.0-compat.patch

```diff
@@ -0,0 +1,76 @@
+diff --git a/cert-to-efi-hash-list.c b/cert-to-efi-hash-list.c
+index 8a5468a..cca2c48 100644
+--- a/cert-to-efi-hash-list.c
++++ b/cert-to-efi-hash-list.c
+@@ -122,6 +122,7 @@ main(int argc, char *argv[])
+ 	       timestamp.Month, timestamp.Day, timestamp.Hour, timestamp.Minute,
+ 	       timestamp.Second);
+ 
++#if OPENSSL_VERSION_NUMBER < 0x10100000L
```

Comment:
Is this upstream patch or…? Also would be nice to cover LibreSSL here as well. @flappyports to the rescue?

Reply:
It was supposted to be. According to the ticket, Quentin Minster is still talking to them and there will me maybe a new version of that patch.

So here is my proposal: I would drop this patch for now, until there is a new version. Instead, we would only do a simple version bump. As Quentin Minster also asked, to proxy maintain, I would step back and let him to do that, if that's okay for you?